scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -119,6 +119,11 @@
                       'Optional (defaults to True). If true, metrics will be collected from the container and reported  '
                       'to Scalyr.', convert_to=bool, default=True)
 
+define_config_option( __monitor__, 'verify_k8s_api_queries',
+                      'Optional (defaults to True). If true, then the ssl connection for all queries to the k8s API will be verified using '
+                      'the ca.crt certificate found in the service account directory. If false, no verification will be performed ',
```

Comment:
Add in a comment saying this can be used to get around issues with older k8 releases.